Tutorial3478/DetailSong/PopUpControllerViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  PopUpControllerViewController.swift
+//  Tutorial3478
+//
+//  Created by Duy Thái on 16/12/2022.
+//
+
+import UIKit
+
+class PopUpControllerViewController: UIViewController {
+
+    @IBOutlet weak private var closeButton: UIButton!
+    @IBOutlet weak private var popUpView: UIView!
+
+    @IBOutlet weak private var noticeLabel: UILabel!
+    private var newNotice = ""
+    
+    override func viewDidLoad() {
+            super.viewDidLoad()
+            configView()
+            self.showAnimate()
+        }
+    
+    private func configView() {
+        self.view.backgroundColor = UIColor.black.withAlphaComponent(0.8)
+        popUpView.layer.cornerRadius = 12
+        closeButton.layer.cornerRadius = 10
+        noticeLabel.text = newNotice
+    }
+        
+    func setNotice(notice: String) {
+        self.newNotice = notice
+    }
+
+    private func showAnimate() {
+    self.view.transform = CGAffineTransform(scaleX: 1.3, y: 1.3)
+    self.view.alpha = 0.0;
+    UIView.animate(withDuration: 0.25, animations: {
+        self.view.alpha = 1.0
+        self.view.transform = CGAffineTransform(scaleX: 1.0, y: 1.0)
+        })
+    }
+
+    @IBAction func closePopUp(_ sender: Any) {
+        self.view.removeFromSuperview()
+    }
+    
+    func removeAnimate() {
```

Comment:
format code toàn bộ file này đi em (Command hoặc Ctrl + I thì phải :v anh không nhớ cái nào)